<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Test-method Suite" duration-ms="15466" started-at="2017-04-23T20:01:24Z" finished-at="2017-04-23T20:01:40Z">
    <groups>
    </groups>
    <test name="Test-method test" duration-ms="15466" started-at="2017-04-23T20:01:24Z" finished-at="2017-04-23T20:01:40Z">
      <class name="com.cgi.code.testng.ExcelTestData">
        <test-method status="PASS" signature="BeforeClass()[pri:0, instance:com.cgi.code.testng.ExcelTestData@34033bd0]" name="BeforeClass" is-config="true" duration-ms="904" started-at="2017-04-23T17:01:24Z" finished-at="2017-04-23T17:01:25Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- BeforeClass -->
        <test-method status="FAIL" signature="DoTest(java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.cgi.code.testng.ExcelTestData@34033bd0]" name="DoTest" duration-ms="13267" started-at="2017-04-23T17:01:25Z" data-provider="name" finished-at="2017-04-23T17:01:39Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Regression]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[login into on Amazon.ca]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[yes]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[ ]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected not same]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected not same
	at org.testng.AssertJUnit.fail(AssertJUnit.java:59)
	at org.testng.AssertJUnit.failSame(AssertJUnit.java:352)
	at org.testng.AssertJUnit.assertNotSame(AssertJUnit.java:310)
	at org.testng.AssertJUnit.assertNotSame(AssertJUnit.java:319)
	at com.cgi.code.testng.ExcelTestData.DoStep(ExcelTestData.java:324)
	at com.cgi.code.testng.ExcelTestData.DoTest(ExcelTestData.java:141)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- DoTest -->
        <test-method status="PASS" signature="TearDown()[pri:0, instance:com.cgi.code.testng.ExcelTestData@34033bd0]" name="TearDown" is-config="true" duration-ms="1248" started-at="2017-04-23T17:01:39Z" finished-at="2017-04-23T17:01:40Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TearDown -->
      </class> <!-- com.cgi.code.testng.ExcelTestData -->
    </test> <!-- Test-method test -->
  </suite> <!-- Test-method Suite -->
</testng-results>
